syft/generic/frameworks/hook/hook.py

```diff
@@ -371,24 +371,63 @@ def overloaded_native_method(self, *args, **kwargs):
             """
 
             if not hasattr(self, "child"):  # means that it's not a wrapper
+
+                # if self is a natural tensor but the first argument isn't,
+                # wrap self with the appropriate type and re-run
+                if len(args) > 0 and hasattr(args[0], "child"):
```

Comment:
hasattr(args[0], "child") .  -> args[0].is_wrapper ?

Reply:
Agreed

Reply:
sometimes args[0] is an int

Reply:
so gotta do hasattr